lib/src/step/generic_step.dart

```diff
@@ -9,24 +9,37 @@ class GenericStep implements BddStep {
     this.rawLine,
     this.testerType,
     this.customTesterName,
+    this.hadDataTable,
   );
 
   final String rawLine;
   final String methodName;
   final String testerType;
   final String customTesterName;
+  final bool hadDataTable;
 
   @override
-  String get content => '''
+  String get content {
+    var result = '';
+    if (hadDataTable) {
+      result +=
+          "import 'package:bdd_widget_test/src/data_table.dart' as bdd;\n";
+    }
+    return result += '''
```

Comment:
The less states (local variables) we put in methods the better. Please use plain old string for this operation:
```suggestion
  @override
  String get content =>
      '${hadDataTable ? "import 'package:bdd_widget_test/src/data_table.dart' as bdd;\n" : ''}'
      '''
import 'package:flutter_test/flutter_test.dart';
```